src/shell/components/FontAwesomeIcon.tsx

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+import React = require('react');
+
+export class FontAwesomeIcon extends React.Component<{ path?: string }, {}> {
```

Comment:
I could see this component being a specialization of a more generic SVG component if/when we actually get custom icons as opposed to just borrowing more from FontAwesome.